nix/pkgs/core.nix

```diff
@@ -16,6 +16,9 @@ rec {
     export NUM_JOBS=8
     export CARGO_BUILD_JOBS=8
 
+    # check all
+    cargo check --all-features --all-targets
```

Comment:
@steveeJ - instead of adding this, what about adding `--all-features` to the `--no-run` cargo tests below?

> `-` `cargo test --no-run --all-targets --manifest-path=crates/holochain/Cargo.toml --features slow_tests,test_utils,build_wasms,db-encryption -- --nocapture --test-threads 1`
> `+` `cargo test --no-run --all-features --all-targets --manifest-path=crates/holochain/Cargo.toml -- --nocapture --test-threads 1`

> `-` `cargo test --no-run --all-targets --workspace --exclude holochain -- --nocapture --test-threads 1`
> `+` `cargo test --no-run --all-features --all-targets --workspace --exclude holochain -- --nocapture --test-threads 1`

If the feature change causes rebuilds during the actual `cargo test` runs, then it probably doesn't save anything over just doing the `cargo check` here as you've done.

Reply:
I don't think this will prevent a rebuild but it seems worth making anyway

Reply:
how about just this instead of the current and the proposed change?

```
diff --git a/crates/holochain/Cargo.toml b/crates/holochain/Cargo.toml
index cdf51cca5..e4d013462 100644
--- a/crates/holochain/Cargo.toml
+++ b/crates/holochain/Cargo.toml
@@ -126,6 +126,7 @@ test_utils = [
   "holochain_state/test_utils",
   "holochain_types/test_utils",
   "holochain_zome_types/test_utils",
+  "kitsune_p2p/test_utils",
   "kitsune_p2p_types/test_utils",
   "itertools",
   "matches"
```